solr/.classpath

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<classpath>
```

Comment:
We would like to keep the repo clean of project files from any particular IDE. Please create a .gitignore file instead of committing this file and the next ones.